clients/clearly-defined/src/funTest/kotlin/ClearlyDefinedServiceFunTest.kt

```diff
@@ -37,6 +37,7 @@ import kotlinx.serialization.json.decodeFromStream
 import org.ossreviewtoolkit.clients.clearlydefined.ClearlyDefinedService.ContributionInfo
 import org.ossreviewtoolkit.clients.clearlydefined.ClearlyDefinedService.ContributionPatch
 import org.ossreviewtoolkit.clients.clearlydefined.ClearlyDefinedService.Server
+import org.ossreviewtoolkit.utils.ort.okHttpClient
```

Comment:
Mind sharing what benefits / impacts setting the user agent properly has, for the changed code? (and the read timeout)

Reply:
> Mind sharing what benefits / impacts setting the user agent properly has

ClearlyDefined can allow a higher rate limit / not block us due to too many requests if they can identify us.

> Does it also impact the connection pool?

No that I'm aware of.